gsap.d.ts

```diff
@@ -822,35 +822,11 @@ declare namespace GSAPStatic {
   const gsap: gsap;
 }
 
-
-
-
-
-
-
-
-
-
-declare const gsap: GSAPStatic.gsap;
-declare const TweenLite: typeof GSAPStatic.Tween;
-declare const TweenMax: typeof GSAPStatic.Tween;
-declare const TimelineLite: typeof GSAPStatic.Timeline;
-declare const TimelineMax: typeof GSAPStatic.Timeline;
-
 declare module "gsap" {
-
-  const gsap: GSAPStatic.gsap;
-  const TweenLite: typeof GSAPStatic.Tween;
-  const TweenMax: typeof GSAPStatic.Tween;
-  const TimelineLite: typeof GSAPStatic.Timeline;
-  const TimelineMax: typeof GSAPStatic.Timeline;
-
-  export {
-    gsap,
-    gsap as default,
-    TweenLite,
-    TweenMax,
-    TimelineLite,
-    TimelineMax
-  }
+  export const gsap: GSAPStatic.gsap;
+  export const TweenLite: typeof GSAPStatic.Tween;
```

Comment:
Given your own suggestion, I would also add a Tween and a Timeline version of the export since the Lite and Max versions are only for backwards compatibility anyway and we want to encourage people to stop using them.

```suggestion
  export const Tween: typeof GSAPStatic.Tween;
  export const TweenLite: typeof GSAPStatic.Tween;
```

Reply:
I tested it and it seems it's not exactly the same. When we export a const like this in a declaration file, we are saying we export an instance of Tween / Timeline. Not the class itself. I still need to use `typeof Tween` in my project if I want to use it as a type.

But types appart, if you do that, I think you also have to export Tween in addition to `export { Tween as TweenMax, Tween as TweenLite }` in your gsap-core.js file. Other way, the declaration file will "lie".